client/actions/tests/test_show-heartbeat.js

```diff
@@ -195,13 +197,23 @@ describe('ShowHeartbeatAction', () => {
       revision_id: 42,
       arguments: {
         surveyId: 'my-survey',
+        includeTelemetryUUID: true,
       },
     });
     const action = new ShowHeartbeatAction(normandy, recipe);
 
     await action.execute();
+
+    // userId should be set
+    expect(normandy.userId).not.toBeNull();
+
+    // userId should be uuid4
+    const UUID_ISH_REGEX = /^[a-f0-9-]{36}$/;
+    expect(UUID_ISH_REGEX.test(normandy.userId)).toBe(true);
```

Comment:
this was following conventions used in other tests - I've gone ahead and made the change throughout them all